zephyr/module.yml

```diff
@@ -0,0 +1,2 @@
+samples:
+  - zephyr/projects
```

Comment:
This is only needed when you want to build this sample as part of the main tree's CI. No need for this.

Reply:
Part of what we need to test with this repository is that every setting in `module.yml` is supported by the Zephyr build system when the module itself is outside of the default Zephyr workspace tree.  Remember, this needs to be an example of how each build system extension mechanism can be used.